packages/honeycomb-opentelemetry-web/src/web-vitals-autoinstrumentation.ts

```diff
@@ -88,11 +88,18 @@ interface VitalOpts extends ReportOpts {
   applyCustomAttributes?: ApplyCustomAttributesFn;
 }
 
-interface VitalOptsWithTimings extends VitalOpts {
+interface LcpVitalOpts extends VitalOpts {
+  /**
+   * Will send the values of these data attributes if they appear on an LCP event
+   */
+  dataAttributes?: string[];
```

Comment:
This pattern could be useful if we wanted to add in the ability to collect `data-` from nodes where attributes aren't available and we need to do a dom lookup. Maybe this is a way to let users opt-into something that might impact "performance"?

Reply:
Ooo. default to sending all and filter down? I like that better for this instance.